Remora.Discord.Commands/Responders/InteractionResponder.cs

```diff
@@ -174,6 +173,7 @@ IApplicationCommandInteractionData commandData
             }
             else
             {
+                var singleOption = options.Single();
```

Comment:
This would miss parameterless commands - use SingleOrDefault and check for null.

Reply:
Parameterless commands have worked for me but I still added a check just in case.

Reply:
This wasn't exactly what I meant - now, if there are no options, all branches will be missed. I'm interested in the fact that parameterless commands were working for you - to me, it shouldn't when you look at this branching. I'll merge this, but won't push it live until I've implemented some unit tests and figured out what the hell I've actually written here x)

Reply:
Parameterless commands probably hit this path
https://github.com/wtfblub/Remora.Discord/blob/72e2209f130aa419187279a27a1ae45595387e32/Remora.Discord.Commands/Responders/InteractionResponder.cs#L141-L144

Reply:
Quite right - I've refactored the paths a bit and added unit tests for all cases, just to be on the safe side. Fixed version should be up on nuget soon.

You should add yourself to the AUTHORS file, by the way :P